pubsub/tests/system.py

```diff
@@ -181,6 +183,8 @@ def test_subscribe_to_messages_async_callbacks(
     # Okay, we took too long; fail out.
     assert callback.calls >= 2
 
+    subscription.close()
+
```

Comment:
Should we be exposing this as a context manager?  Seems like a bug magnet to force explicit closure.

Reply:
This was just a "make sure that close() works". You don't have to call it (all threads are daemon threads).